libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -66,10 +66,9 @@ class AP_InertialSensor_Backend
     virtual AuxiliaryBus *get_auxiliary_bus() { return nullptr; }
 
     /*
-      return the product ID
+     * Return the unique identifier for this backend: it's the same for
+     * several sensors if the backend registers more gyros/accels
```

Comment:
ok, so are we just missing some code for the PX4 backend to get the full ID?
The problem is if we push this in now with the IDs all being 0 and 1 on PX4 boards and then fix the IDs to use the full device IDs later then it will end up forcing all PX4 users to recal